libs/s25main/ingameWindows/iwStatistics.cpp

```diff
@@ -263,15 +290,13 @@ void iwStatistics::Draw_()
             continue;
 
         if(activePlayers[i])
+        {
             DrawRectangle(Rect(drawPt, Extent(34, 12)), player.color);
+            SmallFont->Draw(drawPt + Extent(34, 12 + 2) / 2, getPlayerStatus(player),
```

Comment:
Can you maybe introduce a variable for `Extend(34, 12)` used in both calls as they need to match, don't they?

I guess something like:
```suggestion
            SmallFont->Draw(rect.getOrigin() + rect.getSize() / 2 + DrawPoint(0, 1), getPlayerStatus(player),
```

This should also make it obvious that a (0,1)-offset is intended.